### PR TITLE
Releasing rancher-backups for Rancher v2.7

### PR DESCRIPTION
**Pull request by eliyamlevy**

### CODE REVIEW

File: index.yaml [hunk 1]
```diff
@@ -2637,6 +2637,36 @@ entries:
...
+  - annotations:
+      catalog.cattle.io/auto-install: rancher-backup-crd=match
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/display-name: Rancher Backups
+      catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.25.0-0'
+      catalog.cattle.io/namespace: cattle-resources-system
+      catalog.cattle.io/os: linux
+      catalog.cattle.io/permits-os: linux,windows
+      catalog.cattle.io/provides-gvr: resources.cattle.io.resourceset/v1
+      catalog.cattle.io/rancher-version: '>= 2.6.0-0 < 2.7.0-0'
```
Comment (prachidamle) [inline index.yaml:2649]: This annotation should be updated to 
catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'

**Review by prachidamle — DISMISSED** [review]
rancher version annotation should be: 
catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'

File: packages/rancher-backup/rancher-backup/package.yaml [hunk 2]
```diff
@@ -1,3 +1,2 @@
-url: https://github.com/rancher/backup-restore-operator/releases/download/v2.1.4-rc1/rancher-backup-2.1.4-rc1.tgz
-version: 3.0.0
-doNotRelease: true
+url: https://github.com/rancher/backup-restore-operator/releases/download/v3.0.0-rc1/rancher-backup-3.0.0-rc1.tgz
+ersion: 3.0.0-rc1
```
Comment (prachidamle) [inline packages/rancher-backup/rancher-backup/package.yaml:2]: typo in the key `version`

**Review by prachidamle — APPROVED** [review]

**Comment by eliyamlevy** [comment]
[38983](https://github.com/rancher/rancher/issues/38983)

**Pull request merged**